timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimelockNamespaces.java

```diff
@@ -57,7 +57,7 @@ public final class TimelockNamespaces {
 
     private static final SafeLogger log = SafeLoggerFactory.get(TimelockNamespaces.class);
 
-    private final ConcurrentMap<String, TimeLockServices> services = new ConcurrentHashMap<>();
```

Comment:
Increasing initial capacity to reduce contention

![image](https://github.com/palantir/atlasdb/assets/54594/2d0e8054-1856-49f0-ac26-c1fdc7e0bf0c)

Reply:
From https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/util/concurrent/ConcurrentHashMap.java#L348-L349

> Lock contention probability for two threads accessing distinct elements is roughly 1 / (8 * #elements) under random hashes.

Reply:
Could we spin this specific change into its own PR?

Reply:
Extracted to https://github.com/palantir/atlasdb/pull/7117